openbas-api/src/main/java/io/openbas/rest/executor/ExecutorApi.java

```diff
@@ -143,6 +145,13 @@ public Executor registerExecutor(
       produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
   public @ResponseBody ResponseEntity<byte[]> getOpenBasAgentExecutable(
       @PathVariable String platform, @PathVariable String architecture) throws IOException {
+    if (!AVAILABLE_PLATFORMS.contains(platform)) {
```

Comment:
Given the string collections are constructed with case-lowered names, is it ok to require callers to use lower case path parts as well?

Reply:
I made the changes